lib/glueby/internal/wallet/abstract_wallet_adapter.rb

```diff
@@ -83,10 +83,11 @@ def list_unspent(wallet_id, only_finalized = true)
         #
         # @param [String] wallet_id - The wallet id that is offered by `create_wallet()` method.
         # @param [Tapyrus::Tx] tx - The transaction will be signed.
-        # @param [Array] prevtxs array of hash that represents unbroadcasted transaction outputs used by signing tx. 
+        # @param [Array] prevtxs - array of hash that represents unbroadcasted transaction outputs used by signing tx.
         #                Each hash has `txid`, `vout`, `scriptPubKey`, `amount` fields.
+        # @param [Integer] sighashtype - The sighash flag for each signature that would be produced here.
         # @return [Tapyrus::Tx]
```

Comment:
Please add that `InvalidSighashType` will occur.

Reply:
Added at ed4f4b3